TravelPoints-UI/src/app/app.routes.ts

```diff
@@ -1,7 +1,11 @@
 import {RouterModule, Routes} from '@angular/router';
 import {NgModule} from "@angular/core";
+import {HomePageComponent} from "./components/home-page/home-page.component";
+
+export const routes: Routes = [
+  {path: 'home', component: HomePageComponent},
+];
```

Comment:
S: here you should add the default path, because if any other path is tried, instead of the existing ones, the router will redirect to the default path: `{path: '', component: HomePageComponent}`